### PR TITLE
Enum python

### PR DESCRIPTION
**Pull request by jburel**
# What this PR does

Use constants from enum

# Testing this PR

Check that the build is green and the tests are green

# Related reading

https://trello.com/c/w9GnPkQ7/72-tidy-up-enum-string-constants
Related to https://github.com/openmicroscopy/openmicroscopy/pull/5260

### CODE REVIEW

**Comment by mtbc** [comment]
Should `omero.model.enums.*` be used where the Blitz API is available?

**Comment by jburel** [comment]
probably
still few things to check, wanted to check the full test suite

**Comment by jburel** [comment]
synchonized value for sha1 with https://github.com/openmicroscopy/openmicroscopy/pull/5260

File: components/tools/OmeroPy/src/omero/util/script_utils.py [hunk 1]
```diff
@@ -30,8 +30,11 @@
 from numpy import add, array, asarray, fromstring, reshape, zeros
 from os.path import exists
 
+
```
Comment (mtbc) [inline components/tools/OmeroPy/src/omero/util/script_utils.py]: These *double*-blanks are interesting, are they now a standard way we separate our-code imports?

Comment (jburel) [reply components/tools/OmeroPy/src/omero/util/script_utils.py]: this might have been introduced when moving import around
I will remove it

Comment (mtbc) [reply components/tools/OmeroPy/src/omero/util/script_utils.py]: :+1: I think there was at least another double-blank introduced. Apart from those, this looks good, we'll see when CI is green.

**Review by mtbc — APPROVED** [review]

**Pull request merged**